content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: "cloud"
+page_title: "Managing Variables - Terraform Cloud and Terraform Enterprise"
+description: "Configure Terraform input variables and environment variables and create reusable variable sets that apply to multiple workspaces."
+
+---
+
+# Managing Variables
+
+You can set variables specifically for each workspace or you can create variable sets to reuse the same variables across multiple workspaces. Refer to the [variables overview](/docs/cloud/workspaces/variables.html) documentation for more information about variable types, scope, and precedence.
+
+-> **Note:** Variable sets are in beta.
+
+You need [`read and write variables` permissions](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to create and edit both workspace-specific variables and variable sets. You can manage variables and variable sets through:
+
+- The Terraform Cloud UI, as detailed below.
+- The Variables API for [workspace-specific variables](/docs/cloud/api/workspace-variables.html) and [variable sets](/docs/cloud/api/variable-sets.html).
+- The `tfe` provider's [`tfe_variable`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) resource, which can be more convenient for bulk management.
+
+## Workspace-Specific Variables
+
+To view and manage a workspace's variables, go to the workspace and click the **Variables** tab.
+
+The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace. This is where you can add, edit, and delete workspace-specific variables. You can also apply and remove variable sets from the workspace.
+
+![Screenshot: The initial appearance of a workspace's variables page](./images/vars.png)
+
+
+### Add a Variable
+
+To add a variable:
+
+1. Go to the workspace **Variables** page and click **+ Add variable** in the **Workspace Variables** section.
+
+2. Choose a variable type (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable key, value, and optional description.
+
+    Refer to [variable values and format](#variable-values-and-format) for variable limits, allowable values, and formatting.
+
+3. Click **Save variable**. The variable now appears in the list of the workspace's variables and Terraform Cloud will apply it to runs.
+
+  ![Screenshot: A variable being edited](./images/vars-edit.png)
+
+### Edit a Variable
+
+To edit a variable:
+
+1. Click the ellipses next to the variable you want to edit and select **Edit**.
+2. Make any desired changes and click **Save variable**.
+
+
+### Delete a Variable
+
+To delete a variable:
+
+1. Click the ellipses next to the variable you want to delete and select **Delete**.
+2. Type the variable name into the dialog box to confirm and click **Delete**.
+
+## Variable Sets
```

Comment:
This is a really good comment. Here are my thoughts on this. 
- I agree that this should have a section on the organization-settings page. Thank you - that's SUCH a good catch! 
- I think that adding all of this UI content will make that page even longer than it already is and may overwhelm users. So I think what I'm going to do is create a section that talks about variable sets at a high level and then links to the documentation about how to create and manage them. This matches a pattern I see in some of the other sections under organization settings (for example, teams and VCS providers). It also keeps variable sets content together with all of the content about workspace variables. This is such a weird one because while they are an organizational resource, they are applied to workspaces. So, I think it still makes sense to talk about variable sets in context with workspace variables.

Let me know if you completely disagree - open to more feedback! But this is how I'm thinking about it and I think it works as long as we link to the content in appropriate locations (like organization settings!).